### PR TITLE
(maint) Prep for release 0.5.0 [skip ci]

### PR DESCRIPTION
**Pull request by branan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.5.0] - 2016-04-18
+
+### Fixed
+- Static dependency libraries will no longer be linked to Leatherman consumers
```
Comment (MikaelSmith) [inline CHANGELOG.md]: Only when building Leatherman as shared libraries.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+- Interfacing with ruby APIs for 64-bit integers on Windows
```
Comment (MikaelSmith) [inline CHANGELOG.md]: This is a breaking change, as it removes rb_num2long and rb_num2ulong. They were dangerously inconsistent to use on Windows.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+
+### Added
+- Ability to spawn child processes in a new group on Windows
+- Ruby bindings for `is_bignum`
```
Comment (MikaelSmith) [inline CHANGELOG.md]: Also added a num2size_t Ruby helper.

**Comment by branan** [comment]
@MikaelSmith How's it look now?

**Pull request merged**